src/promptflow-core/promptflow/contracts/tool.py

```diff
@@ -455,3 +455,13 @@ class ToolFuncCallScenario(str, Enum):
     GENERATED_BY = "generated_by"
     REVERSE_GENERATED_BY = "reverse_generated_by"
     DYNAMIC_LIST = "dynamic_list"
+
+
+to_json_type_mapping = {
```

Comment:
Is this only used for assistant tool? If so,  we may not put it in the public contracts file to avoid expose it to user.